src/vsr/replica.zig

```diff
@@ -1312,6 +1313,10 @@ pub fn ReplicaType(
             }
         }
 
+        pub fn invariants(self: *const Replica) void {
```

Comment:
I think we typically call this sort of  function `verify`, but `invariants` is a good name too.